### DISCUSSION TITLE
Unable to connect to wallet

### DISCUSSION THREAD

**Discussion by Juggernaunt22** [Chia-Network/chia-blockchain#4593] [Support]
When trying to connect to my wallet using my 24 word phrase code, I cannot complete the process. The fist step is confirmed (meaning phrase is accepted). You then are taken to the "Restore Metadata for Colored Coins and other Smart Wallets from Backup" screen, I don't have any coins, so I select the "SAFE TO SKIP" . After that a string is displayed  and I select "OK". Next a small box appears that says "fingerpint" and I select "OK". It then just goes back to the same screen and the fingerprint box appears again.

What should I do? How do I fully connect back to my wallet and thus my plots and farming?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
